### ISSUE TITLE
Do we need backend onion services

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe-cli#47]
We have them for the legacy client. Do we need them? [My take](https://github.com/ooni/probe-cli/pull/46/files#r288946922) is the following:

> It's not super clear to me what benefits an onion service would add to our users compared to the current situation where the engine already knows how to proxy any non measurement traffic (including mlab-ns) through any proxy that speaks socks5 (hence also including tor) and uses such proxy service to somehow reach our services exiting on the plain old internet and using https.

**Comment by hellais** [comment]
I agree with you. Having support for onion service may be a nice to have feature for security reasons, but doesn't actually help us much in terms of circumvention and/or protecting users-privacy.

I think we can maybe put this as a low priority nice to have feature, but not something we should focus on working on in the near future.
